### PR TITLE
Docs: fix broken link to readme

### PR DESCRIPTION
**Pull request by cogito-cmurphy**
Just fixing the link in the documentation that is no longer working.

Thanks!

### CODE REVIEW

**Review by mumoshu — APPROVED** [review]
LGTM. Good catch. Thanks a lot for finding and fixing it!

**Comment by cogito-cmurphy** [comment]
thanks @mumoshu , I don't have write so ill leave this up to you. Thanks again!

**Comment by mumoshu** [comment]
@cogito-cmurphy I was just waiting for the CI to pass. Thanks for the reminder 😄

**Pull request merged**
